app/services/umrdr/ldap_config.rb

```diff
@@ -0,0 +1,73 @@
+module Umrdr
+  class LdapConfig
+
+    # class variables from config set in /config/initializers/mcommunity.rb
+
+    # getters
+    def self.host
+      @@host
+    end
+
+    def self.port
+      @@port
+    end
+
+    def self.encryption
+      @@encryption
+    end
+
+    def self.use_ssl
+      @@use_ssl
+    end
+
+    def self.username
+      @@username
+    end
+
+    def self.password
+      @@password
+    end
+
+    def self.auth
+      @@auth
+    end
+
+    # setters
+    def self.set_host(value)
```

Comment:
This should be `self.host=` to follow ruby conventions.  The same for all the setters.